src/components/people/People.tsx

```diff
@@ -0,0 +1,120 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonLink } from '../person/Person';
+
+export const People: React.FC = () => {
+  const [listPeople, setListPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const { slug = '' } = useParams();
+
+  useEffect(() => {
+    getPeople()
+      .then((res: Person[]) => setListPeople(res))
+      .catch(() => setError(true));
+  }, []);
+
+  const parent = (person:Person) => {
+    return listPeople
+      .filter(
+        (el: Person) => el.name === person.fatherName
+          || el.name === person.fatherName,
```

Comment:
```suggestion
        (el: Person) => el.name === person.fatherName
```

Reply:
i need make link father and mather

Reply:
but you are checking absolutely the same things